packages/desktop-client/src/components/accounts/Header.jsx

```diff
@@ -130,6 +130,33 @@ export function AccountHeader({
     },
     [searchInput],
   );
+  useHotkeys(
+    't',
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏just curious - why "t"?

Personally when I think of "create new" operation - I think of "ctrl + n".

Reply:
Yes but that is bound to the browser and a common shortcut for browsers, so we should avoid it.

Reply:
ctrl + t - new tab
ctrl + n - new window

A plain `n` works, but then it overlaps with the notes shortcut and could be confusing.

Reply:
`a` overlapped as well.

Reply:
I wanted `+` personally.  But I can't get the code to accept that as a shortcut.  And others seemed to dislike that one.

Reply:
We're already overriding a bunch of browser defaults, no?

ctrl+f - find (browser) or focus on searchbox (actual)
ctlr+s - save (browser) or sync (actual)
ctrl+p - print (browser) or privacy mode (actual; this would be new)
etc.

Reply:
I think the first two are sane.  This is the 'find' and 'save' functionality of the app, so it makes sense.

Good point on the ctrl-p.  That is one we should think about.

But ctrl-t/ctrl-n (and their +shift variants) I think should *never* be overridden (as well as ctrl-w).

Reply:
I *will* point out that the printing behavior in this app is pretty bad. I do not think people will be printing things.

But it is still a very commonly known/used shortcut, and could be problematic to override.

Reply:
tbh the more I think about it - what you're saying makes sense. :+1:

Reply:
I could see reports having print functionality in the future, so not using ctrl-p could be a good thing

Reply:
I am quite open to suggestions on the privacy toggle.

SHIFT+ctrl+p?

Reply:
> SHIFT+ctrl+p

I cant think of anything better.  crtl-h(idden) is taken.  ctrl-i(ncognito) would probably be confusing.  Its likely not going to be used as much as the other shortcuts so its probably ok that its a longer one.

Reply:
LGTM!